### PR TITLE
Adding proper descriptions in readme.md files for functional tests.

### LINKED ISSUE #1537: Add Readme.md to functional test websites that give brief descriptions of what the website does.

### PR DESCRIPTION
**Pull request by harshgMSFT**
Please click on rich view icon to review.
Fixed #1537

### CODE REVIEW

File: test/WebSites/ActionResultsWebSite/wwwroot/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ActionResultsWebSite
+===
+
+This ASP.NET Web Site illustrates how to use out of box ActionResults which are provided as methods on the controller
```
Comment (yishaigalatzer) [inline test/WebSites/ActionResultsWebSite/wwwroot/readme.md]: Not necessary to say ASP.NET, and use web site (lower caps)

**Comment by yishaigalatzer** [comment]
:shipit: when comments are addressed

**Comment by yishaigalatzer** [comment]
@harshgMSFT we need the readme.md in one folder up, and keep something in the wwwroot folder (so it doesn't get removed). say `HelloWorld.Html` with the content `Hello World`

**Comment by harshgMSFT** [comment]
106b9fc30c33c4c165f0b7bbaf8b314cb02876c4

**Comment by harshgMSFT** [comment]
will send a separate one for helloworld.html

**Pull request closed without merging**